mne_features/univariate.py

```diff
@@ -257,6 +258,62 @@ def compute_kurtosis(data):
     return stats.kurtosis(data, axis=ndim - 1, fisher=False)
 
 
+def compute_rms(data):
+    """Root-mean squared value of the data (per channel).
+
+    Parameters
+    ----------
+    data : ndarray, shape (n_channels, n_times)
+
+    Returns
+    -------
+    output : ndarray, shape (n_channels,)
+
+    Notes
+    -----
+    Alias of the feature function: *rms*
+    """
+    return np.sqrt(np.mean(np.power(data, 2), axis=-1))
+
+
+def compute_prct(data, q=75):
+    """Percentile of the data (per channel).
+
+    Parameters
+    ----------
+    data : ndarray, shape (n_channels, n_times)
+
+    q : int or list
+        Percentile or sequence of percentiles to compute, which must be between
+        0 and 100 inclusive.
+
+    Returns
+    -------
+    output : ndarray, shape (n_channels * len(q),)
+
+    Notes
+    -----
+    Alias of the feature function: *prct*
+    """
+    return np.ravel(np.percentile(data, q, axis=-1))
```

Comment:
actually I would prefer to use the word "quantile" with https://numpy.org/devdocs/reference/generated/numpy.quantile.html

I find it less ambiguous about what it does.

so compute_quantile 

wdyt?

Reply:
Yes, that makes a lot of sense @agramfort. I'll change `percentile` for `quantile`.